webp/view/src/main/cpp/webp_decode.cpp

```diff
@@ -111,6 +112,7 @@ void WebpDecoder::DecodeFrameInternal() {
             break;
         default:
             assert( 0 );
+            delete  [] buf;
```

Comment:
Wait, if you're asserting this won't happen anyway then why not just `abort()`? This whole function is pretty suspicious. Most of these assertions failing mean something has gone horribly wrong and we cannot continue, but at best you just return after them and at worst continue as if nothing is wrong (`assert` will not abort in release builds unless you're explicitly passing `-UNDEBUG` when compiling, but you probably don't want to do that).